### PR TITLE
add servlet

### PR DESCRIPTION
**Pull request by dmuelle**
#5832

### CODE REVIEW

File: modules/ROOT/pages/jakarta-ee10-diff.adoc [hunk 1]
```diff
@@ -134,10 +135,138 @@ If this attribute set to `false`, which is the default, then an archive that con
...
+Alternatively, you can set attributes from the application at request time, as shown in the following example.
+
+[source,java]
+----
+SessionCookieConfig scc = getServletContext().getSessionCookieConfig();
+scc.setName("MyRequestCookieName");
+scc.setAttribute("MyRequestAttribute", "RequestValue");
+----
+
+If conflicts arise among different sources for the cookie configuration, the following precedence order applies, from lowest to highest precedence, to resolve the conflict.
```
Comment (pmd1nh) [inline modules/ROOT/pages/jakarta-ee10-diff.adoc:192]: @dmuelle  should this be "highest to lowest precedence" ?  ( the request time has highest priority; web.xml has lowest priority)

File: modules/ROOT/pages/jakarta-ee10-diff.adoc [hunk 2]
```diff
@@ -134,10 +135,138 @@ If this attribute set to `false`, which is the default, then an archive that con
...
+cookie1: name1
+cookie2: $Path
+cookie3: $Domain
+cookie4: $NAME2
+cookie5: Domain
+-----
+
+=== Application debug enhancements
+
+New `jakarta.servlet.ServletRequest` APIs and new a new `jakarta.servlet.ServletConnection` class are available to help you debug requests to your application. With the enhancement, you can track or refer to a request or obtain details of the network connection that is used by the request by using the request ID.
```
Comment (pmd1nh) [inline modules/ROOT/pages/jakarta-ee10-diff.adoc:231]: remove additional new in (new a new)

File: modules/ROOT/pages/jakarta-ee10-diff.adoc (hunk 1 above)
Comment (dmuelle) [reply modules/ROOT/pages/jakarta-ee10-diff.adoc:192]: thanks for the catch- I misread this in the issue comment. will fix

Comment (dmuelle) [reply modules/ROOT/pages/jakarta-ee10-diff.adoc:192]: fixed in https://github.com/OpenLiberty/docs/pull/5941

File: modules/ROOT/pages/jakarta-ee10-diff.adoc (hunk 2 above)
Comment (dmuelle) [reply modules/ROOT/pages/jakarta-ee10-diff.adoc:231]: fixed in https://github.com/OpenLiberty/docs/pull/5942

FYI this PR was my initial commit and I have not proofread/edited it yet, It probably has  multiple errors that will get fixed when I review the whole page. I will post in the issue when the draft is ready for review- though I do appreciate the catch ;)

**Pull request merged**